cli.js

```diff
@@ -62,19 +65,43 @@ if (!tasks || Object.keys(tasks).length < 1) {
 	process.exit(0);
 }
 
+// get package.json descriptions value
+if (argv.descriptions) {
+	try {
+		descriptions = { exports: {} };
+		require("pkginfo")(descriptions, { dir: cwd, include: ["ntl"] });
```

Comment:
now that `require("pkginfo")` is used in multiple places we could as well have a

 `const pkgInfo = require("pkginfo");` at the top :blush: